src/main/java/org/itstep/j2_16/entity/ProductCategory.java

```diff
@@ -0,0 +1,20 @@
+package org.itstep.j2_16.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Data
+@NoArgsConstructor
+@Entity
```

Comment:
Plz, move this annotation before @Data.
I'd like to see hibernate annotations before lombok annotations